testing/iframe.js

```diff
@@ -71,7 +71,7 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
     // on that window.
     window.beforeLoad = function(win) {
       // Flag as being a test window.
-      win.AMP_TEST = true;
+      win.AMP_TEST_IFRAME = true;
```

Comment:
Yeah that's the semantics I mean.
The `getMode` method currently is reading cached mode variable once the mode is set ([see this](https://github.com/ampproject/amphtml/blob/master/src/mode.js#L49)) Shall I update the mode variable every time I set `window.AMP_TEST`? If that's the case, we can just set mode.test to true for every unit test and we don't even need to set `window.AMP_TEST`.
I don't understand the usage of `AMP_TEST_IFRAME`. Why do we need such a flag for each testing iframe?